lib/main.dart

```diff
@@ -7,109 +9,225 @@ void main() {
-          // Column is also a layout widget. It takes a list of children and
-          // arranges them vertically. By default, it sizes itself to fit its
-          // children horizontally, and tries to be as tall as its parent.
-          //
-          // Invoke "debug painting" (press "p" in the console, choose the
-          // "Toggle Debug Paint" action from the Flutter Inspector in Android
-          // Studio, or the "Toggle Debug Paint" command in Visual Studio Code)
-          // to see the wireframe for each widget.
-          //
-          // Column has various properties to control how it sizes itself and
-          // how it positions its children. Here we use mainAxisAlignment to
-          // center the children vertically; the main axis here is the vertical
-          // axis because Columns are vertical (the cross axis would be
-          // horizontal).
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text(
-              'You have pushed the button this many times:',
+          toolbarHeight: size.height / 8,
+          titleSpacing: size.width / 12,
+          title: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            crossAxisAlignment: CrossAxisAlignment.center,
+            children: <Widget>[
+              SizedBox(
+                child: SvgPicture.network(
+                  'https://upload.wikimedia.org/wikipedia/commons/8/89/DiiaLogo.svg',
+                  height: size.height >= size.width
+                      ? size.width / 8
+                      : size.height / 12,
+                  width: size.height >= size.width
+                      ? size.width / 8
+                      : size.height / 12,
+                ),
+              ),
+              const Icon(
+                Icons.qr_code_scanner_rounded,
+                size: 32,
+              ),
+            ],
+          ),
+          backgroundColor: Colors.transparent,
+          elevation: 0),
+      body: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          Expanded(
+            flex: 4,
+            child: SizedBox(
+              child: PageView.builder(
+                itemCount: cards.length,
+                controller: PageController(viewportFraction: 0.85),
+                onPageChanged: (int index) => setState(() => _index = index),
+                itemBuilder: (_, i) {
+                  return Transform.scale(
+                    scale: i == _index ? 1 : 0.9,
+                    child: Card(
+                      color: cardColors[_index],
+                      elevation: 2,
+                      shape: RoundedRectangleBorder(
+                          borderRadius: BorderRadius.circular(10)),
+                      child: Column(
+                        mainAxisAlignment: MainAxisAlignment.spaceAround,
+                        children: [
+                          Padding(

```

Comment:
You can create a custom widget (with fontSize attribute) to avoid code duplication